docs/tools/test_spec_gen.py

```diff
@@ -58,13 +58,8 @@ def _write_title(file: TextIO) -> None:
     Args:
         file (TextIO): test spec file
     """
-    title = "Test Specification"
-    file.write(title + "\n")
-    file.write("=" * len(title) + "\n")
-    file.write("\n")
-
-    file.write("This file lists all test cases' specifications.\n")
-    file.write("\n")
+    file.write("Test Specification\n==================\n\n")
```

Comment:
Make sure no regression on related documents. The test summary and details page are important.